src/style.css

```diff
@@ -0,0 +1,119 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap");
+
+html {
+  font-family: "Roboto", sans-serif;
+
+  --color-button: #00acdc;
+  --color-white: #fff;
+  --color-border: #f3f3f3;
+  --color-text-black: #060b35;
+  --color-text-grey: #616070;
+}
+
+body,
+h2,
+p {
+  margin: 0;
+}
+
+.block {
+  display: flex;
+  margin: 28px;
+  width: 200px;
+}
+
+.card {
```

Comment:
move each BEM block into separate file

Reply:
Fix it 
![image](https://user-images.githubusercontent.com/97547824/207687241-f5742420-7c33-4f03-aad7-7536517bda3a.png)